src/webgpu/api/validation/compute_pipeline.spec.ts

```diff
@@ -3,6 +3,7 @@ createComputePipeline and createComputePipelineAsync validation tests.
 `;
```

Comment:
nit: add a note to this description, mentioning where the entry point tests are located